### PR TITLE
Refactor/revamp html and css

### PR DESCRIPTION
**Pull request by lschnell8**
PR Template:

Is this a fix or a feature?
- [ ] Bug fix
- [ ] Feature
- [x] Refactor

Describe the feature - fix - refactor:
Clean up HTML by removing unnecessary tags and adding more semantic elements

How was this tested?
- [x] open index.html
- [ ] localhost
- [ ] terminal 
 
Where should the reviewer start - test:
index.html and styles.css

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,67 +1,110 @@
...
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="./styles.css">
+  <link href="https://fonts.googleapis.com/css2?family=Mochiy+Pop+P+One&display=swap" rel="stylesheet">
+  <title>What's Cookin'</title>
+</head>
+
+<body>
+  <header>
+    <h1>What's Cookin'</h1>
+    <input placeholder="search..." />
```
Comment (StevieAmb) [inline src/index.html:14]: We need to remember to add some ids.

File: src/index.html [hunk 2]
```diff
@@ -1,67 +1,110 @@
...
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="./styles.css">
+  <link href="https://fonts.googleapis.com/css2?family=Mochiy+Pop+P+One&display=swap" rel="stylesheet">
+  <title>What's Cookin'</title>
+</head>
+
+<body>
+  <header>
+    <h1>What's Cookin'</h1>
```
Comment (Alynn022) [inline src/index.html:13]: I really like how you simplified the header here.

**Review by StevieAmb — APPROVED** [review]
I like the lists in the recipe card views.

**Review by Alynn022 — APPROVED** [review]
I really like the refactor, really cleans up the code and our html skeleton looks really close to our wireframe now! Awesome css refactor as well!

**Pull request merged**